ecms-social-integration/src/main/java/org/exoplatform/ecms/activity/processor/ActivityAttachmentProcessor.java

```diff
@@ -124,7 +124,7 @@ private String getTitle(Node contentNode) throws RepositoryException {
   }
 
   private boolean isQuarantinedItem(Node node) throws RepositoryException {
-    return node.getPath().startsWith("/" + FileDlpConnector.DLP_QUARANTINE_FOLDER + "/");
+    return node.getPath().startsWith("/" + Utils.DLP_QUARANTINE_FOLDER + "/");
```

Comment:
We need to remove this reference too

Reply:
As discussed  in we keep this check, and only rename the variable.